src/app/mapper.service.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { TestBed } from '@angular/core/testing';
+import { MapperService } from './mapper.service';
+import { REPOSITORIES } from './mock-data';
+import { REPO_DATA1, MAPPED_REPO1, REPO_DATA2, MAPPED_REPO2, MAPPED_PREVIEW_REPO1, MAPPED_PREVIEW_REPO2 } from './test-helpers';
+
+describe('MapperService', () => {
+  let service: MapperService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(MapperService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('#mapRepo should map the full details of a single repository', () => {
+    const actualMapped = service.mapRepo(REPO_DATA1);
+    
+    expect(actualMapped).toEqual(MAPPED_REPO1);
+  });
+
+  it('#mapRepo should map a single repository when only the required details are given', () => {
+    const actualMapped = service.mapRepo(REPOSITORIES[0]);
+    
+    expect(actualMapped).toEqual(MAPPED_PREVIEW_REPO1);
+  });
+
+  it('#mapRepos should map the full details of multiple repositories', () => {
+    const actualMapped = service.mapRepos([REPO_DATA1, REPO_DATA2]);
+
+    expect(actualMapped).toEqual([MAPPED_REPO1, MAPPED_REPO2]);
+  });
```

Comment:
This test and the one starting on line 18 both failed for me because the actual datetime values were an hour before the expected. Probably because I'm in central time and you're in eastern and the expected values are hard-coded?

Reply:
Ah, didn't think of that... will update the mock data to be a Date object instead of a string. Thanks!

Reply:
I think I fixed this by setting Jest's timezone to UTC but will you please lmk if it's still failing on your end?

Reply:
Fixed it for me 👍